css/index.css

```diff
@@ -140,7 +210,58 @@ footer nav {
     font-size: 14px;
 }
 
+footer nav a:hover {
+    background: white;
+    color: black;
+    width: 90px;
+    border-radius: 5px;
+}
+
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+article {
+    margin: 20px;
+    margin-top: 0;
+}
+
+.learn-cards {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    border-top: 1px dashed;
```

Comment:
Good job leveraging the cascading of the Cascading Style sheet and keeping the minimal amount of selectors and properties to avoid inheritance complications! This makes working in your code base simple and easy to track.